### PR TITLE
fix unreachable code

### PR DESCRIPTION
**Pull request by spookydonut**
https://github.com/tgstation/tgstation/pull/48143

### CODE REVIEW

**Comment by ghost** [comment]
thank you!

**Review by ghost — DISMISSED** [review]

File: hippiestation/code/modules/shuttle/emergency.dm [hunk 1]
```diff
@@ -224,7 +224,9 @@
 /obj/machinery/computer/shuttle/pod/emag_act(mob/user)
 	if(obj_flags & EMAGGED)
 		return
-	if(SHUTTLE_DISABLED)//MODULE: SHUTTLE TOGGLE
-		return
+#ifdef SHUTTLE_DISABLED //MODULE: SHUTTLE TOGGLE
```
Comment (ghost) [inline hippiestation/code/modules/shuttle/emergency.dm]: That's not a true/false define.

That's a shuttle define. So this won't work.
Neither did the original code, lol.

**Review by ghost — CHANGES_REQUESTED** [review]

File: hippiestation/code/modules/shuttle/emergency.dm (hunk 1 above)
Comment (ghost) [reply hippiestation/code/modules/shuttle/emergency.dm]: it'd be like `if(SSshuttle.emergency.mode == SHUTTLE_DISABLED)`

Comment (spookydonut) [reply hippiestation/code/modules/shuttle/emergency.dm]: ah ok

**Review by ghost — APPROVED** [review]

**Pull request merged**